target/linux/mediatek/dts/mt7622-ruijie-rg-ew3200gx-pro.dts

```diff
@@ -0,0 +1,321 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+#include "mt7622.dtsi"
+#include "mt6380.dtsi"
+
+/ {
+	model = "Ruijie RG-EW3200GX PRO";
+	compatible = "ruijie,rg-ew3200gx-pro", "mediatek,mt7622";
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+		serial0 = &uart0;
```

Comment:
Please add `ethernet0` alias as well because then U-boot will pass down the MAC address to Linux and there is no need to set it up in userspace (`wan` MAC address will still need handling in userspace, of course)

Reply:
@dangowrt Thanks for the suggestion. I tested it and it works well.